scripts/parseJson.js

```diff
@@ -0,0 +1,171 @@
+const loadJsonFile = require('load-json-file');
+const writeJsonFile = require('write-json-file');
+
+loadJsonFile('./scripts/tempConfig.json').then(json => {
+	var moduleSidebarList = []
+	var completeModules = []
+
+    json.forEach(function (module) {
+      var isHeading = module.kind === 'note'
+      moduleSidebarList.push({
+        isHeading: isHeading,
+        name: module.name,
+        hidden: false
+      })
+      if (!isHeading) {
+        var parent = getParent(module.context.file)
+	      completeModules.push({
+	      	name: module.name,
+          description: concatTags(module.description.children[0].children, true),
+	      	parent: parent,
+	      	snippet: getSnippet(module.examples[0]),
+	      	example: getExample(module.examples[0]),
+	      	hasMap: hasMap(module.examples[0]),
+	      	npmName: getNpmName(module.name, parent),
+	      	returns: getReturns(module.returns),
+	      	params: getParams(module.params),
+	      	throws: getThrows(module.throws)
+	      })
+      }
+	
+	})
+	var config = {
+    	sidebar: moduleSidebarList,
+    	modules: completeModules
+    }
+  writeJsonFile('./template/turf-template/src/assets/config.json', config).then(() => {
```

Comment:
Use the `sync` method to make sure your next lines of code are executed in proper order.

Promises are good when doing HTTP requests, but handling local file system stuff I'd simply just use `sync` that way you code is executed line by line.

```js
write.sync(path.join(__dirname, '..', 'template', 'turf-template', 'src', 'assets', 'config.json'), config)
console.log('Done compiling the config file')
```